teeio-validator/include/helperlib.h

```diff
@@ -42,6 +42,9 @@ uint8_t calculate_checksum(uint8_t *table, size_t length);
 bool is_valid_topology_type(uint8_t *type);
 IDE_TEST_TOPOLOGY_TYPE get_topology_type_from_name(uint8_t* name);
 
+bool is_valid_ide_test_category(uint8_t *test_category);
```

Comment:
why use "ide_test"?

can other test use this one?

Reply:
Renamed as is_valid_test_category.